### PR TITLE
PR: sincronizando fork

### LINKED ISSUE #4: Botão de contato no footer
No footer, embaixo do endereço da FCT, deve-se incluir um botão de enviar e-mail e um para efetuar uma ligação telefônica.

Utilizar `mailto:` para o botão de e-mail. Ele deve apontar para o e-mail do CACiC e já ter um assunto preenchido, para facilitar para o usuário.

Utilizar `tel:` para o botão de telefone. Ele deve apontar para algum ramal da FCT.

Sugestão:

### PR DESCRIPTION
**Pull request by Igor-MPereira**
Closes #4

### CODE REVIEW

File: src/lib/styles/global.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 $body-color: #fefedf;
 $background-color: #000;
 
-@import "../node_modules/bootstrap/scss/bootstrap";
+@import '../node_modules/bootstrap/scss/bootstrap';
+@import url('https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css');
```
Comment (Yudi) [inline src/lib/styles/global.scss:5]: Utilizar o pacote [metonym/svelte-bootstrap-icons](https://github.com/metonym/svelte-bootstrap-icons)
`npm i svelte-bootstrap-icons`

File: src/routes/index-components/footer.svelte [hunk 2]
```diff
@@ -15,58 +15,76 @@
...
+            Universidade Estadual Paulista "Júlio de Mesquita Filho"<br />
+            Rua Roberto Simonsen, 305<br />
+            Presidente Prudente - SP<br />
+            19060-900
+          </span>
+          <br />
+          <a role="button" href="mailto:dmc@fct.unesp.br?subject=SECOMPP-2022">
+            <i class="bi bi-envelope fs-4" />
+          </a>
+          &nbsp;
```
Comment (Yudi) [inline src/routes/index-components/footer.svelte:61]: Margin ou padding ao invés de whitespace
image

File: src/routes/index-components/header.svelte [hunk 3]
```diff
@@ -1,34 +1,53 @@
...
+          >19ª Semana do Curso de Ciência da Computação da FCT-Unesp</span>
       </div>
-      <div class="text-center intro-heading" style="font-weight: 300">
-        <span class="fs-4">Junte-se a nós de 17 a 21 de outubro</span>
+      <div class="text-center intro-heading">
+        <span class="fs-4 fw-light">Junte-se a nós de 17 a 21 de outubro</span>
       </div>
+      <!-- TODO: Alterar cor-->
       <!-- TODO: Usar elemento Button? -->
       <a
```
Comment (Yudi) [inline src/routes/index-components/header.svelte:24]: Seria possível/melhor utilizar `<Button>` ao invés de `<a>`, para economizar no CSS?

File: src/routes/index-components/header.svelte [hunk 4]
```diff
@@ -1,34 +1,53 @@
...
+    padding-right: 16px
+    padding-bottom: 8px
+    padding-top: 8px
+
+	.secompp-logo
+	  max-width: 750px
+
+	.header-bg
+	  background: linear-gradient(0deg, black 0%, rgb(33, 32, 13) 100%)
+    padding-top: 50px
```
Comment (Yudi) [inline src/routes/index-components/header.svelte:52]: Indentação incorreta, utilizar chaves {} para evitar isso

File: src/routes/index-components/nav.svelte [hunk 5]
```diff
@@ -37,9 +49,14 @@
...
       <NavItem>
-        <!-- TODO: Remover margin-left no menu colapsado-->
-        <Button style="margin-left: 8px;" outline>Inscrever-se</Button>
+        <Button class="nav-subscribe-btn" outline>Inscrever-se</Button>
       </NavItem>
     </Nav>
   </Collapse>
 </Navbar>
+
+<style lang="sass">
```
Comment (Yudi) [inline src/routes/index-components/nav.svelte:58]:
```suggestion
<style lang="scss">
```

File: src/routes/index-components/header.svelte [hunk 6]
```diff
@@ -1,34 +1,53 @@
...
-        <span class="fs-3" style="font-weight: 600">19ª Semana do Curso de Ciência da Computação da FCT‑Unesp</span>
+        <span class="fs-3 fw-semibold"
+          >19ª Semana do Curso de Ciência da Computação da FCT-Unesp</span>
       </div>
-      <div class="text-center intro-heading" style="font-weight: 300">
-        <span class="fs-4">Junte-se a nós de 17 a 21 de outubro</span>
+      <div class="text-center intro-heading">
+        <span class="fs-4 fw-light">Junte-se a nós de 17 a 21 de outubro</span>
       </div>
+      <!-- TODO: Alterar cor-->
```
Comment (Yudi) [inline src/routes/index-components/header.svelte:22]:
```suggestion
```

File: src/routes/index-components/about.svelte [hunk 7]
```diff
@@ -53,3 +64,9 @@
   </div>
   <hr />
 </div>
+
+<style lang="sass">
```
Comment (Yudi) [inline src/routes/index-components/about.svelte:68]:
```suggestion
<style lang="scss">
```

File: src/routes/index-components/footer.svelte [hunk 8]
```diff
@@ -15,58 +15,76 @@
...
-<style>
-  .footer-row > .col-md-4 {
-    margin-top: 20px;
-  }
-
-  footer {
-    margin-bottom: 24px;
-    margin-top: 32px;
-  }
+<style lang="sass">
```
Comment (Yudi) [inline src/routes/index-components/footer.svelte:71]:
```suggestion
<style lang="scss">
```

File: src/routes/index-components/header.svelte [hunk 9]
```diff
@@ -1,34 +1,53 @@
...
-                padding-bottom: 8px;
-                padding-top: 8px;
-              ">Inscrever‑se</a
-      >
+        href="https://fct-pp.web.app">Inscrever-se</a>
     </div>
   </div>
 </header>
+
+<style lang="sass">
```
Comment (Yudi) [inline src/routes/index-components/header.svelte:32]:
```suggestion
<style lang="scss">
```

File: src/routes/index-components/footer.svelte [hunk 10]
```diff
@@ -15,58 +15,76 @@
...
+            FCT – Faculdade de Ciências e Tecnologia<br />
+          </h3>
+          <span class="fw-light small">
+            Universidade Estadual Paulista "Júlio de Mesquita Filho"<br />
+            Rua Roberto Simonsen, 305<br />
+            Presidente Prudente - SP<br />
+            19060-900
+          </span>
+          <br />
+          <a role="button" href="mailto:dmc@fct.unesp.br?subject=SECOMPP-2022">
```
Comment (Yudi) [inline src/routes/index-components/footer.svelte:58]:
```suggestion
          <a role="button" href="mailto:cacic.fct@gmail.com?subject=Contato SECOMPP22">
```

**Review by Yudi — CHANGES_REQUESTED** [review]

**Comment by Yudi** [comment]
Ótimo trabalho com o CSS :-)

**Comment by Yudi** [comment]
> [CLA assistant check](https://cla-assistant.io/cacic-fct/site-secompp-2022?pullRequest=5) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cacic-fct/site-secompp-2022?pullRequest=5) before we can accept your contribution.You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cacic-fct/site-secompp-2022?pullRequest=5) it.

@Igor-MPereira @ViniCastilho por favor, [assinem os termos de contribuição](https://cla-assistant.io/cacic-fct/site-secompp-2022?pullRequest=5).

**Comment by Igor-MPereira** [comment]
Fiz os ajustes pedidos

**Pull request merged**